<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanTest</class>
 <widget class="QWidget" name="CanTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QComboBox" name="baudrate">
     <item>
      <property name="text">
       <string>10Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>40Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>125Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>400Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>666Kbps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>800Kbsp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000Kbsp</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="edit">
     <property name="text">
      <string>12345678</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="currentBaudrate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Current Baudrate : 10Kbps </string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="indent">
      <number>-2</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="start">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>recv</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="label"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="datasend">
     <property name="text">
      <string>datasend</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="can0">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>can0</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="can1">
     <property name="text">
      <string>can1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
